werkzeug_encryptedcookie/__init__.py

```diff
@@ -21,6 +24,14 @@ def dumps(cls, data):
         # dict -> bytes
         return json.dumps(data, ensure_ascii=False).encode('utf-8')
 
+    @classmethod
+    def compress(cls, data):
+        return brotli.compress(data, quality=8)
+
+    @classmethod
+    def decompress(cls, data):
+        return brotli.decompress(data)
```

Comment:
Change the order of the methods like for all other methods (first encoders chain, then decoders)

Reply:
I accidentally clicked merge button when I was switching windows. And there is no way to revert it(for me). 
Here is a new one: https://github.com/homm/werkzeug-encryptedcookie/pull/6
Sorry.